core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.html

```diff
@@ -28,7 +28,8 @@
     <div class="oppia-login-required" ng-if="$ctrl.userDetailsLoading || $ctrl.userIsLoggedIn">
       <opportunities-list load-opportunities="$ctrl.loadContributions"
                           on-click-action-button="$ctrl.onClickViewSuggestion"
-                          label-required="!$ctrl.activeReviewTab">
+                          label-required="!$ctrl.activeReviewTab"
+                          opportunity-heading-truncation-length="30">
```

Comment:
The width of the row in the "My contribution" tab is shorter and lead to UI issues with lengthy text (gets divided into multiple lines)